src/debugger/BreakpointManager.js

```diff
@@ -68,15 +74,15 @@ export class BreakpointManager {
         if (breakpoint.column !== 0) {
           // this is a column breakpoint
           if (
-            filePath === this._previousStop.filePath &&
-            lineNum === this._previousStop.line &&
-            colNum === this._previousStop.column
+            filePath === this._lastExecuted.filePath &&
```

Comment:
Why do you need to check "breakpoint.column !== 0" here? this._lastExecuted.column can never be zero right? Then, this whole code can be collapsed into:
```js
return filePath === this._lastExecuted.filePath && lineNum === this._lastExecuted.line && breakpoint.column === this._lastExecuted.column ? null : breakpoint;
```

Reply:
This is to distinguish between line and column breakpoints. Line breakpoints will have breakpoint.column === 0.

Reply:
You can still write it much more compact as:
```js
return filePath === this._lastExecuted.filePath && lineNum === this._lastExecuted.line && (breakpoint.column === 0 || breakpoint.column === this._lastExecuted.column) ? null : breakpoint;
 ```